content/html/concepts/tags/terms/acronym/acronym.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '<acronym>'
+Description: 'Creates text when hovered over the text will be displated'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Accessibility'
+  - 'DOM'
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+  
+## `<acronym>` tag
+
+The `<acronym>` was used the make acronyms before it was deprecated. Now use the [`<abbr>`](https://www.codecademy.com/resources/docs/html/tags/abbr) instead. When using the tag there will be a dotted underline for the word to signify that it is an acronym.
+  
+## Syntax
+
+Use the `title` attribue to add the meaning of the acronym but it is not required but nothing will happen when it is hovered over.
+  
+```html
+<p>A great place to learn to code is <acronym title="Codecademy">CC</acronym><p>
+```
+  
+## Example
```

Comment:
This is an `<abbr>` example. It should be an `<acronym>` example, with a comment noting it's depreciated.